tests/test_imports.py

```diff
@@ -7,3 +7,4 @@ def test_basic(self):
         import cleverhans
         import tensorflow_gcs_config
         import tensorflow_datasets
+        import segment_anything
```

Comment:
I can't really write a fast smoke test because the smallest model weights file is 375Mb.